src/strawberryRunnerUtilityService.php

```diff
@@ -298,19 +318,91 @@ public function invokeProcessorForAdo(ContentEntityInterface $entity, array $sbf
               }
             }
           }
+          // JSON/Metadata level plugins coming from ADO JSON directly , $plugin_definition['input_type'] == entity:node
```

Comment:
@alliomeria this is code for the future. This here will allow Post processors to also act on pure NODES and their metadata. I have no plugin that needs this but once we have a few ideas this can used directly.